src/voicegroup.ts

```diff
@@ -1,24 +1,32 @@
 // [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
 
+import { ModifierContext } from './modifiercontext';
 import { Vex } from './vex';
+import { Voice } from './voice';
 
 /** @constructor */
```

Comment:
This comment does not make sense

Reply:
It was already there this is jsdoc. I wanted to check what is produces with typedoc. The comments I normally rearrange on a second step to avoid that diff stops working.

Reply:
Yes, this is from the pre-ES6 JS days when there was no notion of a constructor. You can remove this.

Reply:
@0xfe removing the comment I have just realised that this class `voicegroup` has no parent. Is that ok or should it extend `element`?

Reply:
This is intentional. They're just containers for formatting logic.

Reply:
Then this one is ready.